src/user/introduction.rst

```diff
@@ -8,7 +8,7 @@ A brief introduction
 
 What is conda-forge?
 --------------------
-Conda-forge can be defined as a compilation of distributions, recipes and build infrastructure that makes the task of installing conda packages hassle-free for a wide range of software. It is a single channel that provides all the up-to-date, compatible versions of conda packages along with all the required dependencies for the conda package manager.
+The conda-forge project can be viewed as a compilation of distributions, recipes, and build infrastructure that enable installing and distributing a wide range of software as conda packages. In a single channel, the conda-forge community provides up-to-date, compatible versions of conda packages along with all their required dependencies. Our work is driven by the community to provide packaged software free of charge and in a transparent manner.
```

Comment:
Should be "enables", since the subject of this phrase is "compilation".

Reply:
Could go either way ;) in the original edit, I thought the "distributions, recipes, and build infrastructure" are the enabling ones

(but actually, I rephrased it precisely to avoid dealing with the stupid collective nouns, which I hate dealing with LOL :D https://www.britannica.com/dictionary/eb/qa/Collective-Nouns-and-Verb-Agreement)

Reply:
Your call @kathatherine

Reply:
I get it, but the compilation of them is still doing the enabling. :) 

I agree that collective nouns are the worst, though. Go ahead and change it, please, @shreya024.

Reply:
Changed